cmd/profile/add/add.go

```diff
@@ -175,18 +176,24 @@ func LoadProfilesFromFile(filePath string) ([]models.DeviceProfile, error) {
 	}
 
 	var profiles []models.DeviceProfile
-
-	//check if the file contains just one DeviceProfile
 	var p models.DeviceProfile
-	err = json.Unmarshal(file, &p)
+	//First check if the file contains Yaml
+	err = yaml.Unmarshal(file, &p)
 	if err != nil {
-		//check if the file contains list of DeviceProfile
-		err = json.Unmarshal(file, &profiles)
+		//check if the file contains just one DeviceProfile
```

Comment:
comment does not match code

Reply:
Could I address this comment in my follow-up PR about update DP . This code will be common for both create and update DP.